lib/faraday/rack_builder.rb

```diff
@@ -198,19 +202,26 @@ def dup
     # :ssl - Hash of options for configuring SSL requests.
     def build_env(connection, request)
       Env.new(request.method, request.body,
-              connection.build_exclusive_url(request.path, request.params, request.options.params_encoder),
+              connection.build_exclusive_url(
+                request.path, request.params,
+                request.options.params_encoder
+              ),
               request.options, request.headers, connection.ssl,
               connection.parallel_manager)
     end
 
     private
 
+    LOCK_ERR = "can't modify middleware stack after making a request".freeze
```

Comment:
nit: double quotes aren't needed here, and single quotes seem to be the preference in the rest of the code.

Reply:
It's because of the `'` in `can't`, the other solution is to use single quotes but escape that single quote